### DISCUSSION TITLE
What's an atomic rule?

### DISCUSSION THREAD

**Discussion by rmx90210** [pest-parser/pest#1051] [Q&A]
The [Atomic](https://pest.rs/book/grammars/syntax.html#atomic) section of the docs jumps straight into how to create an atomic rule without describing what "atomic" means in the context of using Pest, or what the benefits are. I'm lost as a beginner!

**Comment by tomtau** [comment]
Could be worth opening an issue on https://github.com/pest-parser/book to improve that section?

It's described more in the implicit whitespace sub-section: https://pest.rs/book/grammars/syntax.html#implicit-whitespace

> The optional rules WHITESPACE and COMMENT implement this behaviour. If either (or both) are defined, they will be implicitly inserted at every sequence and between every repetition (except in **atomic rules**).

Maybe a better name than "atomic" would be "explicit whitespace". Does it help?
